### ISSUE TITLE
Moving a plugin with the SDK results in a 

### ISSUE THREAD

**Issue by gelform** [Freemius/wordpress-sdk#27]
I created a new cPanel account and moved a domain that used a plugin that uses Freemius. The files and db did not change.

Once the path changed, I received the error:
file_exsts(): open_basedir restriction in effect.

http://prntscr.com/9zd3a5

**Comment by vovafeldman** [comment]
**Note:** Seems like the plugin's disk path has changed, and due to security restrictions, the following command throws the warning:
```php
if ( file_exists( $this->_storage->plugin_main_file->path ) ) {
```

Need to find a way to check if the PHP process can access the path at all.

**Comment by vovafeldman** [comment]
This was resolved in 1.9.0. When the cached file path isn't found `_find_caller_plugin_file()` purges the cache and finds the new path.
